pages/osx/log.md

```diff
@@ -0,0 +1,20 @@
+# log
+
+> View, export, and configure logging systems.
+> More information: <https://www.dssw.co.uk/reference/log.html>.
+
+- Stream live system logs:
+
+`log stream`
+
+- Stream logs sent to `syslog` from the process with a specific PID:
+
+`log stream --process {{process_id}}`
+
+- Show logs sent to syslog from a process with a specific name:
+
+`log show --predicate "process == '{{process_name}}'"`
+
+- Export all logs to disk for the past hour:
+
+`sudo log collect --last {{1h}} --path {{path/to/file.logarchive}}`
```

Comment:
@marchersimon why `.logarchive` instead of `.log`?
```suggestion
`sudo log collect --last {{1h}} --path {{path/to/file.log}}`
```

Reply:
From the man page:
> -output path
>
> Save the archive to the specified path or file. If the path is a directory, a file named system_logs.logarchive will be created in the specified directory. If the path contains the extension .logarchive, a new logarchive will be created with that name at the specified path.

But I just noticed that I confused `output` with `path`. It should be
```suggestion
`sudo log collect --last {{1h}} --output {{path/to/file.logarchive}}`
```